assets/blocks/lesson-actions/lesson-actions-block/settings.js

```diff
@@ -13,19 +15,46 @@ import { __ } from '@wordpress/i18n';
  * Inspector controls for lesson actions block.
  *
  * @param {Object}        props
- * @param {ToggleBlock[]} props.toggleBlocks Blocks to toggle.
+ * @param {ToggleBlock[]} props.toggleBlocks    Blocks to toggle.
+ * @param {string}        props.previewState    Preview state.
+ * @param {Function}      props.onPreviewChange Preview change callback.
  */
-export const LessonActionsBlockSettings = ( { toggleBlocks } ) => (
-	<InspectorControls>
-		<PanelBody title={ __( 'Additional Actions', 'sensei-lms' ) }>
-			{ toggleBlocks.map( ( block ) => (
-				<ToggleControl
-					key={ block.label }
-					checked={ block.active }
-					onChange={ block.onToggle }
-					label={ block.label }
+export const LessonActionsBlockSettings = ( {
+	toggleBlocks,
+	previewState,
+	onPreviewChange,
+} ) => (
+	<>
+		<BlockControls>
+			<ToolbarGroup>
+				<ToolbarDropdown
+					options={ [
+						{
+							label: __( 'In progress', 'sensei-lms' ),
+							value: 'in-progress',
```

Comment:
I wonder if it would make sense to extract `in-progress` and `completed` to constants, since they are also used elsewhere (e.g. [here](https://github.com/Automattic/sensei/pull/3903/files?diff=split&w=1#diff-37a57197ae8ea2b4a62ef33199387972288ba9b5da829c90f602d998f73b2aacR26))?

Reply:
Yep! Makes totally sense! Thank you for the suggestion! https://github.com/Automattic/sensei/pull/3903/commits/e5dd6e15b9568c407c9dfbb9b23ebc8205cd73f8